listenbrainz/webserver/static/js/src/APIService.ts

```diff
@@ -286,4 +286,18 @@ export default class APIService {
     error.response = response;
     throw error;
   };
+
+  getCoverArt = async (
+    releaseMBID: string,
+    recordingMSID: string
```

Comment:
why do we need both of these parameters?

Reply:
wouldn't just releaseMBID be ok?

Reply:
I dont think we have lookup based on releaseMBID